tests/e2e/templates/velero_bsl_template.tmpl

```diff
@@ -0,0 +1,26 @@
+apiVersion: oadp.openshift.io/v1alpha1
+kind: Velero
```

Comment:
The problem is we have phased out this `kind` from master branch so this PR may need updating.

Reply:
specifically... https://github.com/openshift/oadp-operator/pull/459

Reply:
@kaovilai I have updated the yaml as per new API changes. However, after the test run, I can see, there is failure in some other test, which is not affected by this PR - velero_deployment_suite_test.go:103